src/test-utils/dom/collection-preferences/content-display-preference.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import styles from '../../../collection-preferences/styles.selectors.js';
+import dragHandleStyles from '../../../internal/drag-handle/styles.selectors.js';
+import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+import ToggleWrapper from '../toggle';
+
+const getClassName = (suffix: string): string => styles[`content-display-${suffix}`];
+
+export class ContentDisplayOptionWrapper extends ComponentWrapper {
+  /**
+   * Returns the drag handle for the option item.
+   */
+  findDragHandle(): ElementWrapper {
+    return this.findByClassName(dragHandleStyles.handle)!;
+  }
+
+  /**
+   * Returns the text label displayed in the option item.
+   */
+  findLabel(): ElementWrapper {
+    return this.findByClassName(styles['sortable-item-label'])!;
+  }
+
+  /**
+   * Returns the visibility toggle for the option item.
+   */
+
+  findToggle(): ToggleWrapper {
+    return this.findComponent(`.${styles['sortable-item-toggle']}`, ToggleWrapper)!;
+  }
+}
+
+export default class ContentDisplayPreferenceWrapper extends ComponentWrapper {
+  static rootSelector = styles['content-display'];
+
+  /**
+   * Returns the title.
+   */
+  findTitle(): ElementWrapper {
+    return this.findByClassName(getClassName('title'))!;
+  }
+
+  /**
+   * Returns the preference description displayed below the title.
+   */
+  findDescription(): ElementWrapper {
+    return this.findByClassName(getClassName('description'))!;
+  }
+
+  /**
+   * Returns the options that the user can reorder.
+   */
+  findOptions(): Array<ContentDisplayOptionWrapper> {
```

Comment:
Normally we also add a method to find option by index (1-based).

Reply:
Thanks, done in [b8c9fe9](https://github.com/cloudscape-design/components/pull/804/commits/b8c9fe97d4ef83a8d87ee0b38d210e4d7fbb26a2)